pennylane/ops/qubit.py

```diff
@@ -1601,6 +1603,60 @@ def _matrix(cls, *params):
         return U
 
 
+class ControlledQubitUnitary(QubitUnitary):
+    r"""ControlledQubitUnitary(U, control_wires, wires)
+    Apply an arbitrary fixed unitary to ``wires`` with control from the ``control_wires``.
+
+    **Details:**
+
+    * Number of wires: Any (the operation can act on any number of wires)
+    * Number of parameters: 1
+    * Gradient recipe: None
+
+    Args:
+        U (array[complex]): square unitary matrix
+        control_wires (Sequence[int] or int): the control wire(s)
+        wires (Sequence[int] or int): the wire(s) the unitary acts on
```

Comment:
* `QubitUnitary` doesn't have `Wires` here, but think that might be because its docstring was made before `Wires`
* Having no `Wires` might confuse users who look at the docs, maybe good to add?
```suggestion
        control_wires (Union[Wires, Sequence[int], or int]): the control wire(s)
        wires (Union[Wires, Sequence[int], or int]): the wire(s) the unitary acts on
```